README.md

```diff
@@ -21,67 +21,163 @@ e.g.
+
+Activate Cekit and run the following command
+```console
 $ make bundle-dev
+```
+
+the last log line is something like this:
+```console
+INFO  Image built and available under following tags: quay.io/<your_quay_username>/rhpam-operator-bundle:7.12.1, quay.io/${USERNAME}/rhpam-operator-bundle:latest
+```
+**Push the bundle on the container registry**
+
+VERSION=$(go run getversion.go)
+
+USERNAME=<your_quay_username>
+```console
+$ docker push quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION}
+```
+
+#### Build the index
+```console
+opm index add --bundles quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION} --tag quay.io/${USERNAME}/rhpam-operator--index:${VERSION} --permissive
+```
+#### Push the index on the container registry
+Podman images
+```console
+podman push quay.io/${USERNAME}/rhpam-operator--index:${VERSION}
+```
 
+#### Disable default catalog sources on Openshift
+To test your Operator, with bundle and index you need to disable the default source like the operator hub
+```console
+oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
+```
 
-$ docker push ${BUNDLE}
-BUNDLE_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ${BUNDLE})
-INDEX_VERSION=v4.7
-INDEX_IMAGE=quay.io/${USERNAME}/ba-operator-index:${INDEX_VERSION}
-INDEX_FROM=${INDEX_IMAGE}_$(go run getversion.go -csvPrior)
-INDEX_TO=${INDEX_IMAGE}_${VERSION}
+#### catalog-source.yaml
+```console
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: xxxxxname
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: xxxxximage
+  displayName: My Operator Catalog
+  publisher: grpc
+```
 
-$ opm index add -c docker --bundles ${BUNDLE_DIGEST} --from-index ${INDEX_FROM} --tag ${INDEX_TO}
+Choose a CATALOG_SOURCE_NAME something like "max-operator-manifests"
 
-$ docker push ${INDEX_TO}
+```console
+sed -i "s/xxxxxname/$CATALOG_SOURCE_NAME/" catalog-source.yaml
+```
+```console
+sed -i "s/xxxxximage/$(echo $INDEXING_IMAGE | sed "s.\/.\\\/.g")/" catalog-source.yaml
+```
 
-# only run in dev env
-$ oc patch operatorhub.config.openshift.io/cluster -p='{"spec":{"disableAllDefaultSources":true}}' --type=merge
-$ oc apply -f - <<EOF
+Example of catalog-source
+```console
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: my-catalog
+  name: max-operator-manifests
   namespace: openshift-marketplace
 spec:
-  displayName: "Dev Bundles"
-  publisher: "Red Hat"
   sourceType: grpc
-  image: ${INDEX_TO}
-  updateStrategy:
-    registryPoll:
-      interval: 45m
-EOF
+  image: quay.io/<your_quay_id>/rhpam-operator--index:7.12.1
+  displayName: My Operator Catalog
+  publisher: grpc
 ```
 
-It will take a few minutes for the operator to become visible under the _OperatorHub_ section of the OpenShift console _Catalog_. 
+#### Create catalog source on Openshift
+```console
+oc create -f catalog-source.yaml
+```
+
+#### subscription.yaml
```

Comment:
It may not be clear for what is subscription needed, please add more comments.
There can be also mentioned, that you can use Operator Hub to list your operators and create subscriptions.